embedded-graphics/src/primitives/circle.rs

```diff
@@ -24,23 +25,23 @@ use crate::style::WithStyle;
 /// # let mut display = MockDisplay::default();
 ///
 /// // Default circle with only a stroke centered around (10, 20) with a radius of 30
-/// let c1 = Circle::new(Point::new(10, 20), 30);
+/// let mut c1 = Circle::new(Point::new(10, 20), 30);
 ///
 /// // Circle with styled stroke and fill centered around (50, 20) with a radius of 30
-/// let c2 = Circle::new(Point::new(50, 20), 30)
+/// let mut c2 = Circle::new(Point::new(50, 20), 30)
 ///     .stroke_color(Some(Rgb565::RED))
 ///     .stroke_width(3)
 ///     .fill_color(Some(Rgb565::GREEN));
 ///
 /// // Circle with no stroke and a translation applied
-/// let c3 = Circle::new(Point::new(10, 20), 30)
+/// let mut c3 = Circle::new(Point::new(10, 20), 30)
 ///     .stroke_color(None)
 ///     .fill_color(Some(Rgb565::BLUE))
 ///     .translate(Point::new(65, 35));
 ///
-/// display.draw(c1);
-/// display.draw(c2);
-/// display.draw(c3);
+/// c1.draw(&mut display);
+/// c2.draw(&mut display);
+/// c3.draw(&mut display);
```

Comment:
Could these be moved onto the ends of the `Circle::new()` chains above? The `let mut`s could be removed then as well.